scheduler/src/test/scala/com/sky/kms/base/KafkaIntSpecBase.scala

```diff
@@ -4,26 +4,39 @@ import com.sky.kms.kafka.Topic
 import com.sky.kms.utils.RandomPort.randomPort
 import eu.timepit.refined.auto._
 import net.manub.embeddedkafka.Codecs.{nullDeserializer, stringDeserializer}
-import net.manub.embeddedkafka.ConsumerExtensions.{ConsumerRetryConfig, _}
-import net.manub.embeddedkafka.{Consumers, EmbeddedKafka, EmbeddedKafkaConfig}
+import net.manub.embeddedkafka.{EmbeddedKafka, EmbeddedKafkaConfig}
 import org.apache.kafka.clients.consumer.{ConsumerRecord, KafkaConsumer}
 import org.apache.kafka.common.serialization.Deserializer
 import org.scalatest.WordSpecLike
 
-trait KafkaIntSpecBase extends EmbeddedKafka with WordSpecLike with Consumers {
+import collection.JavaConverters._
+import scala.concurrent.duration._
+import scala.compat.java8.DurationConverters._
+
+trait KafkaIntSpecBase extends EmbeddedKafka with WordSpecLike {
```

Comment:
```suggestion
trait KafkaIntSpecBase extends EmbeddedKafka { this: PatienceConfiguration =>
```

`WordSpecLike` is not needed and I'd rather introduce `PatienceConfiguration` than having a copy/paste of `timeout: FiniteDuration = 60.seconds` all over.